bhv/cnative/platform_compatibility.h

```diff
@@ -0,0 +1,19 @@
+
+// IEEE754 Floating Point Access
+#if __APPLE__
+
+    //Little Endian 32-bit float
+    union ieee754_float {
```

Comment:
I don't get this paradigm; all the code used from ieee754.h is duplicated here. Why keep the include for Linux?

Reply:
Since Linux has a header for it, I think using it makes sense.

The libgcc header handles big endian and additional formats, so it's better than my quick and dirty compatibility definition.

All Macs (Since 2004) use little endian, but that's not true of all linux machines.

Reply:
Just a point on philosophy, you will not find a single productized C library that doesn't have platform-specific includes.  So this idea of a "platform_compatibility" shim include is a common pattern, and we'll likely end up with a bunch of stuff in here eventually.

It's ugly but it's the way it is with C.